control/lti.py

```diff
@@ -395,8 +389,7 @@ def evalfr(sys, x, squeeze=None):
 
     See Also
     --------
-    freqresp
-    bode
+    frequency_response, bode_plot
```

Comment:
Possible to include `Statespace.__call__`and `TransferFunction.__call__` here, and also in `evalfr` below? (Is there also a `__call__` for `frd `systems? )

also - my opinion is that evalfr should be deprecated and only left in the Matlab module. Or given a more pythonic name that serves as a wrapper for call. But in any case, evalfr appears nowhere else in the documentation and should not be referenced in `frequency_response`. but `__call__` should be.